src/dune_rules/action_unexpanded.ml

```diff
@@ -63,19 +70,20 @@ module Action_expander : sig
 
     val prog_and_args : String_with_vars.t -> (Action.Prog.t * string list) t
 
-    (* Expand a template statically and pass it to [f]. Raise if it cannot be
-       expanded statically. *)
-    val static_path : String_with_vars.t -> f:(Path.t -> 'a t) -> 'a t
+    module Early : sig
```

Comment:
Don't we call this the "rule evaluation" stage elsewhere? It seems like we should be consistent and reuse that name. E.g. `When_eval`.

Reply:
Makes sense. `When_eval` seems a bit too general to me, so I used `At_rule_eval_stage`